.cursor/skills/jira-to-pr/SKILL.md

```
@@ -0,0 +1,279 @@
+
+Only after approval, create via GitHub MCP `create_pull_request` or `gh pr create`.
+
+---
+
+## Phase 6 — Link back to JIRA
+
+### >>> PERMISSION GATE: JIRA-WRITE <<<
+
+Prepare a JIRA comment summarizing what was done. Present it:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)